### PR TITLE
#36 changed splashScreen, introScreen logo

### PR DESCRIPTION
**Pull request by CSeal**
**What?** 

#36 Update App with rebranded background images

**Why?**
rebranding
**How?**
changed images for IntroScreen(logo), changed splash screen for Android (need to update it for iOS)
**How to test?**
compare the app with Figma [markup](https://www.figma.com/file/CDpPSEGMegUbDwguymGFmQ/ROH-05%3A-TV-App)

### CODE REVIEW

File: android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <resources>
-    <color name="bootsplash_background">#0E1B31</color>
+    <color name="bootsplash_background">#050F20</color>
```
Comment (tomdyson) [inline android/app/src/main/res/values/colors.xml:2]: Should we be setting colours like this globally, so they can be used by other platforms, e.g. tvOS?

**Comment by sinewave440hz** [comment]
Looks fine, except the button is lower down than in the design: 
image

File: android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (sinewave440hz) [reply android/app/src/main/res/values/colors.xml:2]: The colours for splashes do need to be defined on the native side for the splash screens, but we could call them the same on tvOS and Android and reference them in the app. The advantage being that native theme/contrast adjustments would then also apply in-app, automatically...may be worth considering. https://reactnative.dev/docs/platformcolor.

**Review by sinewave440hz — CHANGES_REQUESTED** [review]
Just requesting the button placement change. Unless it is decided to be unimportant of course...

**Pull request merged**